clar2wasm/src/wasm_generator.rs

```diff
@@ -56,7 +56,7 @@ pub struct WasmGenerator {
     /// The locals for the current function.
     pub(crate) bindings: HashMap<String, Vec<LocalId>>,
     /// Size of the current function's stack frame.
-    frame_size: i32,
```

Comment:
Unrelated?

Reply:
Some methods return u32 like `ListTypeData::get_max_len()`. Although, if something can't be negative shouldn't it be unsigned? I can't see that frame_size is going to be -10 or something.